sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
> Hmm, maybe we could use #3851 to truly solve this?

I think we'd need a pretty good reason to block this further than we already have.

> I think the resetting to $0 option would be safer though...

Agreed. How many users are affected if we go this route? I suspect it's not very many ...